tests/shell/src/script.sh

```diff
@@ -0,0 +1,43 @@
+#!env sh
+# should raise SC2239
+
+# Some bad code taken from https://github.com/koalaman/shellcheck
+echo $1
+rm "~/my file.txt"
+touch $@
+echo 'Path is $PATH'
+[[ n != 0 ]]
+[[ -e *.mpg ]]
+[[ $foo==0 ]]
+[[ -n "$foo " ]]
+[[ $foo =~ "fo+" ]]
+[ foo =~ re ]
+[ $1 -eq "shellcheck" ]
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6196008)